spark/src/main/scala/io/delta/sharing/spark/RemoteDeltaCDFRelation.scala

```diff
@@ -26,7 +26,12 @@ import org.apache.spark.sql.functions.col
 import org.apache.spark.sql.sources.{BaseRelation, Filter, PrunedFilteredScan}
 import org.apache.spark.sql.types.StructType
 
-import io.delta.sharing.spark.model.{CDFColumnInfo, Metadata, Table => DeltaSharingTable}
+import io.delta.sharing.spark.model.{
```

Comment:
It's just code refactoring in RemoteDeltaCDFRelation and RemoteDeltaFileIndex to support both streaming and non-streaming case.